src/composables/useSidebar.ts

```
@@ -39,18 +55,24 @@ interface OpenAPIOperation extends OpenAPIV3.OperationObject {
   'operationId': string
 }
 
-const DEFAULT_CONFIG: Required<Omit<SidebarConfig, 'methodAliases' | 'spec'>> = {
+const DEFAULT_CONFIG: Required<Omit<SidebarConfig, 'methodAliases' | 'spec' | 'sidebarItemTemplate' | 'sidebarGroupTemplate'>> = {
   linkPrefix: '/operations/',
   tagLinkPrefix: '/tags/',
   defaultTag: 'Default',
 } as const
 
+function defaultGroupTemplate(path: string, _: number = 1): string {
```

Biome (1.9.4)

[error] This type annotation is trivially inferred from its initialization.

Safe fix: Remove the type annotation.


(lint/style/noInferrableTypes)